connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/feel/jackson/FeelDeserializer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.runtime.core.feel.jackson;
+
+import com.fasterxml.jackson.databind.BeanProperty;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import io.camunda.connector.runtime.core.feel.FeelEngineWrapper;
+import java.util.Map;
+
+public class FeelDeserializer extends AbstractFeelDeserializer<Object> {
```

Comment:
The only issue I see is that connectors would need to depend on the runtime to make use of it. Maybe move it to the SDK?

Reply:
Right, we will then have to move a few dependencies like feel engine and jackson module scala to the SDK.

Reply:
https://github.com/camunda/connector-sdk/pull/525